key.js

```diff
@@ -111,7 +42,6 @@ const gameModule = (function () {
         const currentLevel = gameState.levelsInfo.currentLevel;
         if (!gameState.levelsInfo.setInfoList[currentLevel] && gameState.lives > 0 && gameState.seconds > 0) {
             resetGameLivesAndTimer();
-            const gameLevel = initialGameState();
             gameState.levelsInfo.currentLevel++;
```

Comment:
`  gameState.levelsInfo.currentLevel += incrementBy;`
Why do we send a parameter on the above Production code? I have another query, what is the value of incrementBy? 
by default it will be undefined if not provided, right? If we send in a value like `incrementGameLevel(4)`
where, `gameState.levelsInfo.currentLevel = 3`, it might make gameState.levelsInfo.currentLevel = 7
which is invalid for a 5 level game. Can you please explain that line?